recipes/openpmd-validator/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "openpmd-validator" %}
+{% set version = "1.1.0.4" %}
+{% set build = 0 %}
+{% set sha256 = "a15153c95350e713866ff44a307548f575bd4a7833b082d65fe2d5d0551d48f8" %}
```

I'm not a big fan of jinja variables for stuff that is used only once. Seems like an unnecessary complication.

---

Oh, I thought the templates create this. Updated!
```suggestion
{% set build = 0 %}
```